packages/shared/src/hooks/useOnPostClick.ts

```diff
@@ -128,7 +128,7 @@ export default function useOnPostClick({
         }
 
         if (!post.read) {
-          await checkReadingStreak();
```

Comment:
This is still async?

Reply:
I removed the async because there weren't any _await_ in `checkReadingStreak`

Reply:
Is it not? `clearQueries` is async?https://github.com/dailydotdev/apps/pull/3736/files#diff-eb59d027316931f66780bf3544a0f792cec29664cea0c29003a1295d11a39516R85

Reply:
No, it isn't a Promise 👀 

this is the implementation:
<img width="834" alt="image" src="https://github.com/user-attachments/assets/6dba6bdc-11da-48f9-8fc4-53c8240c70a8">

Reply:
It's actually a bug in `useDebounceFn` implementation, it should also be async and await if result of a provided function is async. But yeah, not relevant for this case.

Reply:
Btw. no need to fix it here. @ilasw